atd-vze/src/views/Tools/CreateCrashRecordTable.js

```diff
@@ -56,26 +47,25 @@ const CreateCrashRecordTable = ({ crashesData, loading, error }) => {
     setCrashSearch(e.target.value);
   };
 
-  // /**
-  //  * Deletes all the temporary records in the database
-  //  */
-  // const handleDelete = () => {
-  //   setFeedback(null);
-  //   deleteRecord({ variables: { crashId: deleteId } })
-  //     .then(() => {
-  //       const newData = data["crashes"].filter(item => {
-  //         return item.crash_id !== deleteId;
-  //       });
-  //       setDeleteId(null);
-  //       setFeedback(`Crash ID ${deleteId} has been deleted.`);
-  //       setCrashesData({ atd_txdot_crashes: newData });
-  //       toggleModalDelete();
-  //     })
-  //     .catch(err => {
-  //       setFeedback(String(err));
-  //       setDeleteId(null);
-  //     });
-  // };
+  /**
+   * Soft deletes the temporary crash and all its associated unit and people records
```

Comment:
thanks for being more specific about the types of records affected in this comment. the old comment threw me for a loop!